kokoro/release/python/linux/build_artifacts.sh

```diff
@@ -7,11 +7,13 @@ pushd $(dirname $0)/../../../..
 
 export REPO_DIR=protobuf
 export BUILD_VERSION=`grep -i "version" python/google/protobuf/__init__.py | grep -o "'.*'" | tr -d "'"`
-export BUILD_COMMIT=v$BUILD_VERSION
+export BUILD_COMMIT=master
```

Comment:
If we pin to master, we need to change it when we work on release branch?

Reply:
Changed to extract from the KOKORO_JOB_NAME name instead.